src/datadoc/datadoc-json-schema.json

```diff
@@ -213,6 +213,12 @@
                     "description": "The data set contains data up until date/time",
                     "type": "string",
                     "format": "date"
+                },
+                "pseudonymization_time": {
+                    "title": "Dataset pseudo time",
+                    "description": "Time at which the dataset was pseudonymized. In ISO 8601 format.",
+                    "type": "string",
+                    "format": "date-time"
```

Comment:
We should move this field into the variable structure (and update the title and description 😇 )